sdks/java/io/parquet/src/main/java/org/apache/beam/sdk/io/parquet/ParquetIO.java

```diff
@@ -262,19 +264,36 @@ public static Sink sink(Schema schema) {
     @Nullable
     abstract String getJsonSchema();
 
+    @Nullable
+    abstract CompressionCodecName getCompressionCodec();
+
+    abstract Builder toBuilder();
+
     @AutoValue.Builder
     abstract static class Builder {
       abstract Builder setJsonSchema(String jsonSchema);
 
+      abstract Builder setCompressionCodec(CompressionCodecName compressionCodec);
+
       abstract Sink build();
     }
 
+    /** Specifies compression codec. By default, CompressionCodecName.UNCOMPRESSED. */
```

Comment:
The default should be probably SNAPPY to be consistent with AvroIO (as discussed in slack).

Reply:
ok